### PR TITLE
[IMP] Adding alias to multitail and configured multitail.conf

### PR DESCRIPTION
**Pull request by JesusZapata**
Referring to https://github.com/Vauxoo/docker-odoo-image/issues/141#event-931147355

### CODE REVIEW

**Comment by JesusZapata** [comment]
@moylop260 
I don't not understand the point " Consider replace the current colorized encoding of the odoo logs"
In file /etc/multitail.conf I cant't see the secction odoo!

**Comment by moylop260** [comment]
Use a file.log where the colorized char is not exported.
Just is exported the plain txt.

`tail -f file.log` show you without colorized.
`tail2 -f file.log` should show colorized.

Could you add print-screen about?

**Comment by JesusZapata** [comment]
The WARNING and CRITICAL Are not displayed correctly

image

@moylop260 Is that what you mean?

**Comment by moylop260** [comment]
Add a regex to support them

**Comment by JesusZapata** [comment]
@moylop260 
The way it looks correctly is commenting some rules of log4j!
Is this a good idea?

image

The lines are line 624 and 627

image

File: odoo-shippable/scripts/build-image.sh [hunk 1]
```diff
@@ -235,6 +235,26 @@ git_ps1_style(){
...
 EOF
 
+# Add alias for multitail and configure
+cat >> /etc/bash.bashrc << EOF
+alias tail2="multitail -cS log4j"
+EOF
+
+cat >> /etc/multitail.conf << EOF
+# Odoo log
+colorscheme:odoo
```
Comment (moylop260) [inline odoo-shippable/scripts/build-image.sh]: If you are adding a new schema, then this one should be used from `multitail -cS odoo` or not?

Comment (JesusZapata) [inline odoo-shippable/scripts/build-image.sh]: @moylop260 
You're right!
Thanks!

**Comment by JesusZapata** [comment]
@moylop260 
Screenshot

multitail

**Comment by JesusZapata** [comment]
@moylop260 
The travis is red because of an isolated fact
https://travis-ci.org/Vauxoo/docker-odoo-image/builds/195086054#L3886
```
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-QPD_Gl/pyparsing/
You are using pip version 8.1.1, however version 9.0.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
The command '/bin/sh -c bash /usr/share/vx-docker-internal/odoo80/build-image.sh' returned a non-zero code: 1
```

**Pull request merged**